frontends/mit-open/src/pages/ErrorPage/ErrorPage.tsx

```diff
@@ -40,7 +41,9 @@ const ErrorPage = () => {
      * This should not happen, but it's better than the app crashing.
      */
     <ErrorPageTemplate title="Error">
-      <h1>Something went wrong.</h1>
+      <Typography variant="h3" component="h1">
```

Comment:
> Is this for where we want to apply an h3 theme to an h1 element

Yes

> should we be able to bend rules around presentation hierarchy?

Do you mean "is styling h3 as h1 something we should encourage"? IMO, yes: The h1 style in design docs is enormous. Easy for me to imagine pages (like this one) where we want a semantic h1 but we want it styled differently.

Also easy for me to imagine cases where we want something styled as a heading but not semantically a heading at all. (E.g., so that a screenreader doesn't announce it as a heading.)